docs/bigquery_to_vcf.md

```diff
@@ -0,0 +1,116 @@
+TEMP_LOCATION=gs://BUCKET/temp
+
+COMMAND="/opt/gcp_variant_transforms/bin/bq_to_vcf \
+  --project ${GOOGLE_CLOUD_PROJECT} \
+  --input_table ${INPUT_TABLE} \
+  --output_file ${OUTPUT_FILE} \
+  --temp_location ${TEMP_LOCATION} \
+  --job_name bq-to-vcf \
+  --runner DataflowRunner"
+gcloud alpha genomics pipelines run \
+  --project "${GOOGLE_CLOUD_PROJECT}" \
+  --logging "${TEMP_LOCATION}/runner_logs_$(date +%Y%m%d_%H%M%S).log" \
+  --zones us-west1-b \
+  --service-account-scopes https://www.googleapis.com/auth/cloud-platform \
+  --docker-image gcr.io/gcp-variant-transforms/gcp-variant-transforms \
+  --command-line "${COMMAND}"
+```
+
+Besides, the following optional flags are provided:
+* `--representative_header_file`: If provided, meta-information from the
+  provided file will be added into the `output_file`. Otherwise, the
+  meta-information is inferred from the BigQuery schema. To best recover the
+  VCF, it is recommended to provide this file.
+* `--number_of_bases_per_shard`:  The maximum number of base pairs per
+  chromosome to include in a single VCF file (one shard). A shard is a
+  collection of data within a contiguous region of the genome. By default,
+  number_of_bases_per_shard = 1,000,000. With the default setting, the variants
+  on the same chromosome with start position 1 to 999,999 are written to the
+  same shard, and the variants on the same chromosome with start position
+  1,000,000 to 1,999,999 are written to another shard, etc. This parameter will
+  have an impact on memory requirements since the data in a single shard must be
+  sorted.
+* `--genomic_regions`: A list of genomic regions (separated by a space) to load
+  from BigQuery. The format of each genomic region should be
+  CHROMOSOME:START_POSITION-END_POSITION or CHROMOSOME if the full chromosome is
+  requested. Only variants matching at least one of these regions will be
+  loaded. If this parameter is not specified, all variants will be kept.
+* `--call_names`: A list of call names (separated by a space). Only variants for
+  these calls will be loaded from BigQuery. If this parameter is not specified,
+  all calls will be kept.
+* `--allow_incompatible_schema`: If `representative_header_file` is not
+  provided, the meta-information is inferred from the BigQuery schema. There are
+  some reserved fields based on
+  [VCF 4.3 spec](http://samtools.github.io/hts-specs/VCFv4.3.pdf). If the
+  inferred definition is not the same as the reserved definition, an error will
+  raise and the pipeline fails. By setting this para to be true, the
+  incompatibilities between BigQuery schema and the reserved fields will not
+  raise errors. Instead, the VCF meta information are inferred by the schema
+  without validation.
+
+### Running from github
+
+In addition to using the docker image, you may run the pipeline directly from
+source.
+
+Example command for DirectRunner:
+
+```bash
+python -m gcp_variant_transforms.bq_to_vcf \
```

Comment:
This is super cool that you actually used 1000 genomes! We can revisit later to have an example command option for the docker run too.